docs/iris/src/whatsnew/contributions_2.1/incompatiblechange_2018-May-01_remove_experimental_um.txt

```diff
@@ -0,0 +1 @@
+* Removed :mod:`iris.experimental.um` to ease integration of Mule.
```

Comment:
Not sure about the "to ease..." bit.
I think this should instead explain in terms of what to replace with.
Perhaps? :
```"Removed :mod:`iris.experimental.um`.  Please use <https://github.com/SciTools/mule> instead."```

According to @stevewardle, that repo address is likely to be replaced soon by https://github.com/metomi/mule
 -- coming, but not ready yet.

Reply:
@pp-mo yeah, I couldn't think of what to put here – it almost stopped at just "Removed :mod:`iris.experimental.um`". Your suggestion is good though, I'll put it in.